### PR TITLE
Add minutes from 2020-09-02 core meeting

### PR DESCRIPTION
**Pull request by ericdill**

### CODE REVIEW

File: src/orga/minutes/2020-09-02.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+    * **UPDATE 2020-07-22**: We in principle have agreement to render the extra pinnings needed directly in the feedstock 
+      on a temporary basis (i.e., until the migration has ended).
+
+## Discussion
+
+## Check in on previous action items
+Copy previous action items from last meeting agenda.
+
+### This meeting
+* [ ] (ED) Update governance docs with similar voting model as what got put into conda-tools (+3 with no -1 is a pass)
```
Comment (beckermr) [inline src/orga/minutes/2020-09-02.md:124]: this stuff never made the final vote for that governance doc

Comment (ericdill) [reply src/orga/minutes/2020-09-02.md:124]: wait really? i thought that went in

File: src/orga/minutes/2020-09-02.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+
+****
+
+[Zoom link](https://flatiron.zoom.us/j/93242638216?pwd=bjRCWmVJRW1oTGJhN09VUmxtTTJOUT09 )
+[What time is the meeting in my time zone](https://arewemeetingyet.com/UTC/2020-08-26/17:00/w/Conda-forge%20dev%20meeting#eyJ1cmwiOiJodHRwczovL2hhY2ttZC5pby9wUk15dFVKV1FmU3NJM2xvMGlqQzJRP2VkaXQifQ==)
+[last weeks meeting](https://hackmd.io/uoxd_bzSRie5Dq1ZrMSvhA)
+
+## Attendees
+
+- Matt B.
```
Comment (ocefpaf) [inline src/orga/minutes/2020-09-02.md:11]: A lot was done in a meeting with only one person attending ;-p

File: src/orga/minutes/2020-09-02.md (hunk 1 above)
Comment (ericdill) [reply src/orga/minutes/2020-09-02.md:124]: ah looks like it got discussed but never added to the doc?

Comment (beckermr) [reply src/orga/minutes/2020-09-02.md:135]: yup

File: src/orga/minutes/2020-09-02.md (hunk 2 above)
Comment (beckermr) [reply src/orga/minutes/2020-09-02.md:11]: You know how I roll...

**Pull request merged**